### PR TITLE
Add Appendix: Planned Enhancement 

### LINKED ISSUE #359: [PE-D][Tester D] Does not allow for special characters (eg: s/o or d/o) in names
-------------
Labels: `severity.Low` `type.FunctionalityBug`
original: yuvrajaryan/ped#3

### LINKED ISSUE #364: [PE-D][Tester C] Testing for emails is lax
Emails like aa@bb and 123@456 are still accepted in your system. Perhaps, you could consider checking that the emails are in the format of abc@def.xyz to ensure validity even more strictly.

-------------
Labels: `type.FunctionalityBug` `severity.Low`
original: natlv/ped#4

### LINKED ISSUE #390: [PE-D][Tester E] Addappt only allows one appointment to be added at a time (feature improvement suggestion)
As a doctor who prioritises time and efficiency, I would want to schedule multiple appointments at once with a patient. 

The current implementation allows me to only schedule one appointment at a time.

A proposed improvement is as seen below:
"addappt addAppt Dental i/S1234567A @d/2024-10-27 @t/1100-1200 @d/2024-11-27 @t/1100-1200 @d/2024-12-27 @t/1100-1200"

This allows me to schedule back-to-back monthly appointments with my patient without having to retype the same command 3 times.

Hence, the feature seems incomplete as it can be better optimised for the target user. Perhaps the team can consider taking on this suggestion.

-------------
Labels: `severity.Low` `type.FeatureFlaw`
original: ywllowsensor/ped#7

### PR DESCRIPTION
**Pull request by yong-zhuo**
fixes #390 , fixes #364 , fixes #359

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -1088,3 +1088,17 @@ Deleting an existing appointment from a patient
...
+
+## Appendix: Planned Enhancements
+
+Team size: 5
+
+1. **Allow names to include symbols and special characters**: As of the current version of MediBase3, names are restricted to both alphanumerics and spaces only. 
+However, there are people with legal names that include symbols and special characters, such as 'Nagaratnam s/o Suppiah'. In the future, we will loosen the restrictions for names to allow users to input names with any special characters and symbols.
+
+2. **Improve `addAppt` command to allow users to add multiple appointments at once**: Currently, users can only add one appointment at a time using the `addAppt` command. We plan to enhance the `addAppt` command to allow users to add multiple appointments at once.
+This will be useful as it allows doctors to save time by adding multiple appointments for a patient in one go for patients with recurring appointments, instead of typing the same command multiple times.
```
Comment (PresleyChew) [inline docs/DeveloperGuide.md:1101]: change to "allow"

**Review by PresleyChew — APPROVED** [review]
Overall LGTM

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (yadobler) [inline docs/DeveloperGuide.md:1101]: 'patients with recurring appointments' - might be misleading / too specific. can remove

**Review by TechRaj — APPROVED** [review]
LGTM

**Pull request merged**